test/jit/test_complex.py

```diff
@@ -192,3 +191,14 @@ def fn1(a: complex, b: complex):
 
         x, y = 2 - 3j, 4j
         self.checkScript(fn1, (x, y))
+
+    def test_tensor_attributes(self):
```

Comment:
Are there more tensor attributes that will be added in the future?

Reply:
no just these two